guides/hack/08-arrays-and-collections/01-introduction.md

```diff
@@ -39,3 +39,35 @@ Keyset\keys(dict['x' => 1]); // keyset['x']
 
 Vec\map(keyset[1, 2], $x ==> $x + 1); // vec[2, 3]
 ```
+
+## Arrays Cheat Sheet
+
+| Operation| `vec`    | `dict`   | `keyset` |
+|----------|----------|----------|----------|
+| Initialize empty                             | `$v = vec[];`                | `$d = dict[];`                 | `$k = keyset[];`               |
+| Literal                                      | `$v = vec[1, 2, 3];`         | `$d = dict['foo' => 1];`       | `$k = keyset['foo', 'bar'];`   |
+| From Another Container*                      | `$v = vec($container);`      | `$d = dict($keyed_container);` | `$k = keyset($container);`     |
+| Keys from Container*                         | `$v = Vec\keys($container);` | N/A                            | `$k = Keyset\keys($container);`|
+| Add Elements                                 | `$v[] = 4;`                  | `$d['baz'] = 2;`               | `$k[] = 'baz';`                |
+| Bulk Add Elements                            | `$v = Vec\concat($t1, $t2)`  | `$d = Dict\merge($kt1, $kt2)`  | `$k = Keyset\union($t1, $t2)`  |
+| Remove Elements                              | Remove-at-index is unsupported; `$first = PHP\array_shift($v)` and `$last = PHP\array_pop($v)` | `unset($d['baz']);`| `unset($k['baz']);`|
+| Key Existence                                | `C\contains_key($v, 1)`      | `C\contains_key($d, 'foo')`    | `C\contains_key($k, 'foo')`    |
+| Value Existence                              | `C\contains($v, 3)`          | `C\contains($d, 2)`            | N/A                            |
+| Equality (Order-Dependent)                   | `$v1 === $v2`                | `$d1 === $d2`                  | `$k1 === $k2`                  |
+| Equality (Order-Independent)                 | N/A                          | `Dict\equal($d1, $d2)`         | `Keyset\equal($k1, $k2)`       |
+| Count Elements (i.e., length, size of array) | `C\count($v)`                | `C\count($d)`                  | `C\count($k)`                  |
+| Type Signature                               | `vec<Tv>`                    | `dict<Tk, Tv>`                 | `keyset<Tk>`                   |
+| Type Refinement                              | `$v is vec<_>`               | `$d is dict<_, _>`             | `$k is keyset<_>`              |
+| `Awaitable` Consolidation                    | `Vec\gen($v)`                | `Dict\gen($d)`                 | `Keyset\gen($x)`               |
+
+## Arrays Conversion Cheat sheet
+
+Prefer to use Hack arrays whenever possible. When interfacing with legacy APIs that expect older Containers, it may be easier to convert. Here's how:
+
+| Converting | To `Vector`| To `Map`   | To `Set`   | To `varray`| To `darray`|
```

Comment:
Remove varray and darray; they are now equivalent to vec and dict.

Reply:
I was talking with @Wilfred about this -- I think he mentioned that they're not truly 1:1 equivalent so this could be of use? (which is why I didn't delete). Thoughts?

Reply:
will remove